capa/features/extractors/elf.py

```diff
@@ -458,10 +458,12 @@ def strtab(self) -> Optional[bytes]:
         for d_tag, d_val in self.dynamic_entries:
             if d_tag == DT_STRTAB:
                 strtab_addr = d_val
+                break
```

Comment:
or is there a reason to not stop searching after finding an entry here and below?

Reply:
as long as our tests pass, i dont have an issue with this

Reply:
speaking of tests, just added one for this ELF header